### PR TITLE
Q2

### PR DESCRIPTION
**Pull request by aqattan**

### CODE REVIEW

File: Q2.txt [hunk 1]
```diff
@@ -0,0 +1,18 @@
+javascript:
+var a = ["john", "mike", "susan", "matt", "joe", "larry", "ken", "samantha", "jenny", "Sara", "henry"];
+var x=a.length -1;
+while (x>0){
```
Comment (malkhafaji) [inline Q2.txt]: @aqattan  it should be x >= 0 since 0 is a valid element position in the array, otherwise, x > 0 will skip 0

File: Q2.txt [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+	x=x-2;
+	console.log(a[x].toUpperCase());
+	x=x-1;
```
Comment (malkhafaji) [inline Q2.txt]: @aqattan  it seems redundant to do x - 2 then x - 1 in the same iteration. Just print a[x] then do x -= 3

Comment (ssharba) [inline Q2.txt]: this will shift only one position

**Review by ssharba — CHANGES_REQUESTED** [review]

**Comment by malkhafaji** [comment]
This is good to go, but I don't know how to merge this because I don't have write permission

**Pull request closed without merging**